src/Tmds.Ssh/ECDsaPrivateKey.cs

```diff
@@ -0,0 +1,76 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System;
+using System.Buffers;
+using System.Formats.Asn1;
+using System.Numerics;
+using System.Security.Cryptography;
+
+namespace Tmds.Ssh;
+
+sealed class ECDsaPrivateKey : PrivateKey
+{
+    private readonly ECDsa _ecdsa;
+    private readonly Name _ecIdentifier;
+    private readonly Name _curveName;
+    private readonly HashAlgorithmName _allowedAlgorithm;
+
+    public ECDsaPrivateKey(ECDsa ecdsa, Name identifier, Name curveName, HashAlgorithmName allowedAlgorithm) :
+        base([identifier])
+    {
```

Comment:
Can you rename:
`identifier` -> `algorithm`
`_ecIdentifier` -> `_algorithm`
`allowedAlgorithm` -> `hashAlgorithm`
`_allowedAlgorithm` -> `_hashAlgorithm`.

Reply:
Sure